test/new-e2e/utils/e2e/e2e.go

```diff
@@ -57,16 +57,22 @@ const (
 type Suite[Env any] struct {
 	suite.Suite
 
-	stackName string
-	stackDef  *StackDefinition[Env]
+	stackName       string
+	defaultStackDef *StackDefinition[Env]
+	currentStackDef *StackDefinition[Env]
+	firstFailTest   string
```

Comment:
In the code, only the first failing test is relevant because it prevents the environment to be updated. Is there any reasons to keep other fail tests?